packages/react-components/src/components/icons/grafana.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
```

Comment:
is this icon used anywhere?

Reply:
It's used here: https://github.com/questdb/saas-client/blob/master/modules/connection-details/badges.tsx#L6 (so we can refect to this package in Cloud UI as well)

Reply:
ok! i thought the scope of this PR is just feedback dialog, didn't realise you're also extracting icons.

cool tho

Reply:
@argshook That's because one of the icons is used in Cloud's Feedback Dialog, therefore I used it here as well :)